cli/src/peers.rs

```diff
@@ -11,7 +14,15 @@ static SERVAL_NODE_ADDR: OnceCell<SocketAddr> = async_once_cell::OnceCell::new()
 async fn base_url() -> SocketAddr {
     *SERVAL_NODE_ADDR
         .get_or_init(async {
-            maybe_find_peer("SERVAL_NODE_URL")
+            if let Some(override_addr) = std::env::var("SERVAL_NODE_URL")
```

Comment:
I moved the env var check up here so I wouldn't have to repeat it in the two very similar `maybe_find_peer` functions